### PR TITLE
feat(snippet): add `Right Side Cover Art`

### PR DESCRIPTION
**Pull request by Tetrax-10**
Cover art on right side with animation

### CODE REVIEW

**Review by kyrie25 — CHANGES_REQUESTED** [review]

**Comment by theRealPadster** [comment]
Might I suggest naming it "right side cover art"? My first thought when I read the title was that it was updating incorrect cover art.

File: packages/marketplace/src/resources/snippets.ts [hunk 1]
```diff
@@ -89,4 +89,10 @@ export default [
     "code": ".cover-art .cover-art-image {\n    border-radius: 100% !important;\n}",
     "preview": "resources/assets/snippets/circular-album-art.png",
   },
+  {
+    "title": "Right Cover Art",
```
Comment (kyrie25) [inline packages/marketplace/src/resources/snippets.ts:93]:
```suggestion
    "title": "Right Side Cover Art",
```

**Review by kyrie25 — CHANGES_REQUESTED** [review]
Following Padster's suggestion, this is not forced so dw.

**Review by kyrie25 — APPROVED** [review]
LGTM.

**Pull request merged**